spark8t/services.py

```diff
@@ -1117,6 +1127,7 @@ def create(self, service_account: ServiceAccount) -> str:
             KubernetesResourceType.SERVICEACCOUNT,
             service_account.name,
             namespace=service_account.namespace,
+            **{"username": service_account.name},
```

Comment:
**todo:** Given that we re-use `service_account.name`, might as well add 2 vars up on L1125 for:
```python
username = service_account.name
serviceaccount = service_account.id
```

Reply:
Done